products/memgraph/v0.51.0/how_to_guides/import-data.md

```diff
@@ -91,98 +107,133 @@ CREATE ({first_name: "Melissa", last_name: "Doe", number: 2, aliases: ["Mel"]});
 
 ##### Nodes
 
-When importing nodes, several more types can be specified in the header of the CSV file (along with all property types):
+When importing nodes, several more types can be specified in the header of the
+CSV file (along with all property types):
  - `ID`: id of the node that should be used as the node ID when importing relationships
  - `LABEL`: designates that the field contains additional labels for the node
  - `IGNORE`: designates that the field should be ignored
 
-The `ID` field type sets the internal ID that will be used for the node when creating relationships.
-It is optional and nodes that don't have an ID value specified will be imported, but can't be connected to any relationships.
-If you want to save the ID value as a property in the database, just specify a name for the ID (`user_id:ID`).
-If you just want to use the ID during the import, just leave out the name of the field (`:ID`).
-The `ID` field also supports creating separate ID spaces.
-The ID space is specified with the ID space name appended to the `ID` type in parentheses (`ID(user)`).
-That allows you to have the same IDs (by value) for multiple different node files (for example, numbers from 1 to N).
-The IDs in each ID space will be treated as an independent set of IDs that don't interfere with IDs in another ID space.
-
-The `LABEL` field type adds additional labels to the node.
-The value is treated as an array type so that multiple additional labels can be specified for each node.
-The value is split using the array delimiter (`--array-delimiter` flag).
+The `ID` field type sets the internal ID that will be used for the node when
+creating relationships.  It is optional and nodes that don't have an ID value
+specified will be imported, but can't be connected to any relationships.  If
+you want to save the ID value as a property in the database, just specify a
+name for the ID (`user_id:ID`).  If you just want to use the ID during the
+import, just leave out the name of the field (`:ID`).  The `ID` field also
```

Comment:
I would remove one "just" in this sentence. Either "If you want to use the ID during the import, just leave out...", or "If you just want to use the ID during the import, leave out...".

I guess the second one makes more sense in the broader context.